src/Payments.sol

```diff
@@ -188,23 +315,10 @@ contract Payments is
         nonReentrant
         onlyRailParticipant(railId)
         validateRailNotTerminated(railId)
+        settleAccountLockupBeforeAndAfterForRail(railId, true, 0)
     {
         Rail storage rail = rails[railId];
         Account storage payer = accounts[rail.token][rail.from];
-        OperatorApproval storage approval = operatorApprovals[rail.token][
-            rail.from
-        ][rail.operator];
-
-        // Settle account lockup to ensure we're up to date
-        uint256 settledUntil = settleAccountLockup(payer);
-
```

Comment:
Don't need these anymore as allowance tracking and updates now happen in the `settleAccountLockupBeforeAndAfterForRail` modifier. So does the account lockup settlement.